<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CConfigDlg</class>
 <widget class="QDialog" name="CConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>207</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-4</y>
     <width>321</width>
     <height>210</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="General">
    <attribute name="title">
     <string>General</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Network">
    <attribute name="title">
     <string>Network</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>10</y>
       <width>159</width>
       <height>248</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="1">
       <widget class="CKbdEditBox" name="editIP">
        <property name="toolTip">
         <string>IP</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Mask</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="CKbdEditBox" name="editMask">
        <property name="toolTip">
         <string>Mask</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Gate</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="CKbdEditBox" name="editGate">
        <property name="toolTip">
         <string>Gate</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>DNS</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="CKbdEditBox" name="editDNS">
        <property name="toolTip">
         <string>DNS</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="chkDHCP">
        <property name="text">
         <string>Use DHCP</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>164</x>
       <y>10</y>
       <width>151</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>SSID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CKbdEditBox" name="editSSID"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="keyLabel">
        <property name="text">
         <string>Key</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="CKbdEditBox" name="editKey"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="authLabel">
        <property name="text">
         <string>Auth</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmbAuth">
        <item>
         <property name="text">
          <string>OPEN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WEPAUTO</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WPAEAP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WPA2EAP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WPAPSK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WPA2PSK</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="modeLabel">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cmbMode">
        <item>
         <property name="text">
          <string>Disabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Enabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Every hour</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="encLabel">
        <property name="text">
         <string>Enc.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cmbEnc">
        <item>
         <property name="text">
          <string>NONE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WPA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WPA2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WEP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AES</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Time">
    <attribute name="title">
     <string>Time</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>3</y>
       <width>301</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="4" column="0">
       <widget class="QLabel" name="lblUseNTP">
        <property name="text">
         <string>Use NTP</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="chkUseNTP"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblNTPServer">
        <property name="text">
         <string>NTP Server</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="CKbdEditBox" name="editNTPServer"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblTime">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTimeEdit" name="editTime">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblDate">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="editDate">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CKbdEditBox</class>
   <extends>QLineEdit</extends>
   <header>KbdEditBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CConfigDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CConfigDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
